### ISSUE TITLE
Questions about the lambda hyperparameter

### ISSUE THREAD

**Issue by GorkemP** [tomealbuquerque/ordinal-losses#1]
Hi,

Congratulations for this nice work! I am trying to use CO2 and HO2 in my own work. I have two questions at this point:

1. In the paper, lambda term is multiplied with the ordinal loss terms (`neighbor_loss`), however, in the repository it is multiplied with cross-entropy or Entropy loss. It is a difference between the paper and the implementation right?
2. My dataset is large and it is not feasible to tune the lambda in a short time. To begin, can you write the best/default lambda as you did for the omega term?

Thank you.

**Comment by tomealbuquerque** [comment]
Hi Görkem,

First of all, thank you for your interest in our work. 

1.  "...It is a difference between the paper and the implementation right" -> Yes, but it's not critical because you can do **lambda*CE + neighbor_loss** or you can do **CE + (1/lambda)*neighbor_loss** the result will be the same. It depends what lambda you use.

2. "...can you write the best/default lambda as you did for the omega term?" Lambda depends on the data, but in my last experience with a big dataset, the values of 0.1 and 0.01 were the most used values in CO2 and HO2 loss. My suggestion for you is to start with 0.1 and if needed start reducing.

Good luck with your work, any doubt feel free to contact me again!

**Comment by GorkemP** [comment]
Thank you for the answers.
